plugins/common/opcua/input/input_client.go

```diff
@@ -352,6 +355,9 @@ func (o *OpcUAInputClient) InitNodeMetricMapping() error {
 			if node.MonitoringParams.SamplingInterval == 0 {
 				node.MonitoringParams.SamplingInterval = group.SamplingInterval
 			}
+			if node.Browse == false {
+				node.Browse = group.Browse
+			}
```

Comment:
With the above change you can do
```suggestion
			if node.Browse == nil {
				node.Browse = &group.Browse
			}
```